README.md

```diff
@@ -47,145 +68,6 @@ Method | HTTP request | Description
 [**cenreportPost**](docs/DefaultApi.md#cenreportpost) | **POST** /cenreport | Submit symptom or infection report
 [**cenreportTimestampLowerTimestampUpperGet**](docs/DefaultApi.md#cenreporttimestamplowertimestampupperget) | **GET** /cenreport?timestampLower={tsLower}&timestampUpper={tsUpper} | Returns a list of reports generated between a timestamp range
 
-## AWS Infrastructure Setup
```

Comment:
Might be good to keep in case the TF script breaks. We can rename this to 'Manual AWS Infrastructure Setup'

Reply:
Good point - will do.